app/components/portable-infobox-hero-image.js

```diff
@@ -1,74 +0,0 @@
-import {readOnly} from '@ember/object/computed';
```

Comment:
I believe we can't remove it as it can be used for second infobox

Reply:
it is rendered on php side now